README.md

```diff
@@ -132,12 +132,23 @@ You can create a dashboard with the NewRelic interface and export it to JSON for
 Then you can just copy it to a file within your entity type's folder, modify it if needed and refer to it from the definition.yml:
 
 ```yaml
-dashboardTemplates: 
- - ./<dashboardName>.json
+dashboardTemplates:
+ - template: ./<dashboardName>.json
 ```
 
 Note that you can define more than one dashboard under the `dashboardTemplates` field. 
 
+If your entity has different sources you can also provide a rule to show the dashboard only if the condition matches.
```

Comment:
I'd reword to something like this:

```
When an entity type has several telemetry sources, you can provide a different dashboard for each one.  To add a dashboard specific for a telemetry source, add a block in the `dashboardTemplates` section like this:
```

Not completed as it depends on how the yaml looks like!